src/containers/MealsPage/MealsPage.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import RecipeCard from '../../components/Cards/Recipe-Card/Recipe-Card';
```

Comment:
if you do have the old `Recipe-Card` inside `Cards` with the `s` folder delete that